interface/src/components/App/Encrypt.tsx

```diff
@@ -233,98 +233,114 @@ export function Encrypt(): any {
-                            {textToEncrypt != '' && senderAddress != '' ?
-                                <Box>
-                                    <Typography>
-                                        <small>
-                                            est. rewards: {estimatedReward} DBX
-                                        </small>
-                                    </Typography>
-                                </Box> : 
-                                null
-                            }
-
-                            <LoadingButton className="send-btn" 
-                                loading={loading} variant="contained" 
-                                endIcon={ <SendIcon /> }
-                                sx={{ marginLeft: 2, marginTop: 1 }}
-                                onClick={() => encryptText(textToEncrypt, senderAddress)}>
-                                Send another message
-                            </LoadingButton>
-                        </Box>
-                    }
-                </Box>
-            </div>
+                                            />
+                                        )
+                                    })
+                                }
+                            </Box>
+                        </Stack>
+                        <div className='yellow-bar'></div>
+                        <Editor
+                            editorState={editorState}
+                            onEditorStateChange={handleEditorChange}
+                            toolbarClassName="toolbar"
+                            wrapperClassName="wrapper"
+                            editorClassName="editor"
+                            placeholder='Write your message here'
+                        />
+                        <div className="editor-overlay"></div>
+
+                        { messageSessionSentCounter === 0 ?
+                            <Box sx={{ display: "flex", 
+                                alignItems: "end", 
+                                justifyContent: "flex-end", 
+                                flexDirection: "column", 
+                                mr: 1 }}>
+                                {textToEncrypt != '' && senderAddress != '' ?
+                                    <Box>
+                                        <Typography>
+                                            <div className='estimated-reward'>
+                                                <small>
+                                                    est. rewards: {estimatedReward} DBX
+                                                </small>   
+                                            </div>
+                                        </Typography>
+                                    </Box> : 
+                                    null
+                                }
+
+                                <button disabled={textToEncrypt == '' || addressList == []}
```

Comment:
I think it would be fine to display a spinner or something when we are sending a message.